workshop_1.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Welcome to IndieAGI workshop 1!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## By the end of this workshop you will be able to:\n",
+    "* Excecute python code in a jupyter notebook\n",
+    "* Know the importance of each section of code\n",
+    "* Feel more enabled than you were yesterday"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Instructions to run code\n",
+    "<img src=\"./images/run_cell_instructions.png\" style=\"width: 50%; height: auto;\">\n",
+    "\n",
+    "1. Click on the code block you want to run. \n",
+    "2. Press the play button in the toolbar above to exceute the code.  \n",
+    "  \n",
+    "(You can also press ```shift+enter``` to run the code block and move to the next one.)  \n",
+    "\n",
+    "**Task**: Try in on the next cell"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install\n",
+    "%pip install langchain huggingface_hub\n",
+    "\n",
+    "\n",
+    "# Do you have any questions?\n",
+    "# Any feedback if something is making sense or not? \n",
+    "# Type below following the \"#\" symbol:\n",
+    "#\n",
+    "#\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Obtaining Huggingface access token\n",
+    "1. Go to https://huggingface.co/settings/token\n",
+    "2. Create and login to your account\n",
+    "3. Your screen should look something like this at this point:  \n",
+    "   <img src=\"./images/new_access_token_page.png\" style=\"width: 50%; height: auto;\">  \n",
+    "   <sub> **Troubleshoot**: try entering the exact link above again.</sub>\n",
+    "4. Click on \"New token\" button\n",
+    "5. Type in a name and select options to look like this:  \n",
+    "   <img src=\"./images/access_token_options.png\" style=\"width: 30%; height: auto;\">\n",
+    "6. Click generate token\n",
+    "7. Copy the token to your clipboard\n",
+    "8. Run the below cell of code. You should see an input box appear below the cell looking like this:  \n",
+    "   <img src=\"./images/input_box.png\" style=\"width: 30%; height: auto;\">\n",
```

Comment:
This image is rendered very small and without much context. I suggest changing all screenshots to be the same size -- 13" MacBook -- so the student doesn't have to jump around in visual context.